usecases/connect-autonomous-database/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM maven:3.8-openjdk-11 AS MAVEN_TOOL_CHAIN
+COPY pom.xml /tmp/
+RUN mvn -B dependency:go-offline -f /tmp/pom.xml -s /usr/share/maven/ref/settings-docker.xml
+COPY src /tmp/src/
+WORKDIR /tmp/
+RUN mvn -B -s /usr/share/maven/ref/settings-docker.xml package
+
+FROM openjdk:11-jre-slim
```

Comment:
What about using GraalVM?  It has Java goodness baked in it (along with lots of other goodies).  :)

Reply:
Yes. It is the good idea. As it requires, some additional testing, added to backlog.